lib/src/graph_extension.rs

```diff
@@ -1061,6 +1068,16 @@ mod tests {
         assert_eq!(dag.get_parallel_process_nodes(n0), None);
     }
 
+    #[test]
+    fn test_set_dag_id_normal() {
+        let mut dag = Graph::<NodeData, i32>::new();
+        dag.set_dag_id(0);
+
+        for node_i in dag.node_indices() {
+            assert_eq!(dag[node_i].params["dag_id"], 0);
```

Comment:
This line is not executed because dag has no nodes.

Reply:
Add two nodes.